flow/connectors/postgres/cdc.go

```diff
@@ -626,6 +625,24 @@ func PullCdcRecords[Items model.Items](
 	}
 }
 
+func updateConsumedOffset(
```

Comment:
wouldn't need to pass `p.otelManager` if this was a method

Reply:
nice catch, can also remove call to `NewPostgresMetadataFromCatalog`, but will make it just a method for now